vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -238,7 +238,7 @@ def add_new_req(
             local_physical_block_ids=local_block_ids,
             remote_block_ids=kv_transfer_params["remote_block_ids"],
             remote_engine_id=kv_transfer_params["remote_engine_id"],
-            remote_request_id=kv_transfer_params["remote_request_id"],
+            remote_request_id=kv_transfer_params.get("remote_request_id", ""),
             remote_host=kv_transfer_params["remote_host"],
             remote_port=kv_transfer_params["remote_port"],
```

Comment:
I don't think we should require `remote_engine_id`, `remote_block_ids`, `remote_request_id`, `remote_host`, or `remote_port` for a prefill request - it just happens that existing proxy servers set those fields, but I don't think there's any reason to require them

Let's use `.get(..., None)` for all of them - AFAICT all the other fields are None in a prefill request currently

Reply:
thanks, @markmc , I removed the same commit in #30420, Was trying to fix the issue in one PR
Now this PR is cleaned and only focus on the scheduler change, please help to check